src/main/java/io/supertokens/webserver/api/emailpassword/ResetPasswordAPI.java

```diff
@@ -70,12 +72,13 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
         }
 
         try {
+            TenantIdentifier tenantIdentifier = getTenantIdentifier(req);
+            String userId = EmailPassword.resetPassword(tenantIdentifier, super.main, token, newPassword);
 
-            String userId = EmailPassword.resetPassword(this.getTenantIdentifier(req), super.main, token, newPassword);
+            io.supertokens.pluginInterface.useridmapping.UserIdMapping userIdMapping = UserIdMapping.getUserIdMapping(
+                    tenantIdentifier.toAppIdentifier(), userId, UserIdType.SUPERTOKENS);
```

Comment:
this should be UserIdType.ANY.

Please check everywhere for this type of mistake

Reply:
Fixed